### PR TITLE
Objects Automated Test - LPS-156800, LPS-156801, LPS-156807 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-156800
https://issues.liferay.com/browse/LPS-156801
https://issues.liferay.com/browse/LPS-156807

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase [hunk 1]
```diff
@@ -19,9 +19,11 @@ definition {
 		if ("${testPortalInstance}" == "true") {
 			PortalInstances.tearDownCP();
 		}
+
 		else {
 			ObjectAdmin.deleteAllCustomObjectsViaAPI();
 		}
+
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase]: This looks like it was caught by SF? If you need to make SF changes to code that wasn't part of your changes, you can make them in a separate commit with the message "LPS-156807 Auto SF".

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase [hunk 2]
```diff
@@ -19,11 +19,9 @@ definition {
 		if ("${testPortalInstance}" == "true") {
 			PortalInstances.tearDownCP();
 		}
-
 		else {
 			ObjectAdmin.deleteAllCustomObjectsViaAPI();
 		}
-
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase]: It looks like you added the spaces in the previous commit and removed them here. Please remove them from the original commit next time using interactive rebase.

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectCustomViews.macro [hunk 3]
```diff
@@ -258,6 +258,34 @@ definition {
...
+					key_columnOption = "${filterValueList}",
+					locator1 = "ObjectCustomViews#CHECKBOX_COLUMN_OPTION");
+			}
+		}
+
+		Click(
+			key_name = "Save",
+			locator1 = "Button#GENERAL_BUTTON_MODAL_FOOTER");
+	}
+
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectCustomViews.macro:288]: Do we need to extract this into a macro? The code isn't being re-used and you're using tasks to explain what's happening in the test so I don't think we need to create this macro, unless I'm missing something. 

Out of curiosity, is the expected coding practice in Brazil to not use functions at the test case level?

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase [hunk 4]
```diff
@@ -623,6 +623,50 @@ definition {
...
+				filterValue = "Approved,Denied");
+		}
+
+		task ("Assert Filter edited") {
+			ObjectCustomViews.assertVisibleEntriesOnFilters(
+				filterBy = "Workflow Status",
+				filterValue = "Denied");
+		}
+	}
+
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase:669]: Similar to when we create the filter, we should use the filter on the object entries to make sure the changes are applied.

**Review by timpak — CHANGES_REQUESTED** [review]
@andreefarias please see my comments for required changes. You can send a new PR once you're done, I'm bringing my laptop with me so I'll be checking my review queue once a day.

**Pull request closed without merging**